### PR TITLE
chore(docs): 2020.09 release notes

### LINKED ISSUE #4533: Release notes 2020.09
**Describe the issue. What is the expected and unexpected behavior?**
Release notes 2020.09

### PR DESCRIPTION
**Pull request by tlabaj**
**What**: Closes #4533 

**Additional issues**:

### CODE REVIEW

File: packages/react-docs/RELEASE-NOTES.md [hunk 1]
```diff
@@ -4,6 +4,88 @@ section: overview
 releaseNoteTOC: true
 ---
 
+## 2020.09 release notes (2020-07-17)
+- [@patternfly/react-catalog-view-extension@4.5.1](https://www.npmjs.com/package/@patternfly/react-catalog-view-extension/v/4.5.1)
+- [@patternfly/react-charts@6.6.6](https://www.npmjs.com/package/@patternfly/react-charts/v/6.6.0)
```
Comment (redallen) [inline packages/react-docs/RELEASE-NOTES.md:9]: Change to `@patternfly/react-charts@6.6.0`

File: packages/react-docs/RELEASE-NOTES.md [hunk 2]
```diff
@@ -101,6 +184,7 @@ releaseNoteTOC: true
 - Removed call for yarn clean:pf3 fro package.json ([#4356](https://github.com/patternfly/patternfly-react/pull/4356))
 - Fixed no anonymous functions ([#4391](https://github.com/patternfly/patternfly-react/pull/4391))
 - Updates from OUIA audit ([#4401](https://github.com/patternfly/patternfly-react/pull/4401))
+- Updated tag from prerelease-v4 to prerelease ([#4539](https://github.com/patternfly/patternfly-react/pull/4539))
```
Comment (redallen) [inline packages/react-docs/RELEASE-NOTES.md:187]: This is from the 2020.09 release but it's under the 2020.08 section.

File: packages/react-docs/RELEASE-NOTES.md [hunk 3]
```diff
@@ -90,6 +172,7 @@ releaseNoteTOC: true
 - Made APIs more react friendly with hooks ([#4342](https://github.com/patternfly/patternfly-react/pull/4342))
 - Allowed a promise to be returned by the create connector callback ([#4390](https://github.com/patternfly/patternfly-react/pull/4390))
 - Updated to sync with OpenShift Console ([#4317](https://github.com/patternfly/patternfly-react/pull/4317))
+- Created connector drag spec configuration ([#4529](https://github.com/patternfly/patternfly-react/pull/4529))
```
Comment (redallen) [inline packages/react-docs/RELEASE-NOTES.md:175]: This is from the 2020.09 release but it's under the 2020.08 section.

File: packages/react-docs/RELEASE-NOTES.md [hunk 4]
```diff
@@ -4,17 +4,17 @@ section: overview
...
-- [@patternfly/react-icons@4.4.3](https://www.npmjs.com/package/@patternfly/react-icons/v/4.4.3)
-- [@patternfly/react-inline-edit-extension@4.5.23](https://www.npmjs.com/package/@patternfly/react-inline-edit-extension/v/4.5.23)
-- [@patternfly/react-styles@4.4.5](https://www.npmjs.com/package/@patternfly/react-styles/v/4.4.5)
-- [@patternfly/react-table@4.11.3](https://www.npmjs.com/package/@patternfly/react-table/v/4.11.3)
-- [@patternfly/react-tokens@4.5.3](https://www.npmjs.com/package/@patternfly/react-tokens/v/4.5.3)
-- [@patternfly/react-topology@4.4.25](https://www.npmjs.com/package/@patternfly/react-topology/v/4.4.25)
-- [@patternfly/react-virtualized-extension@4.5.15](https://www.npmjs.com/package/@patternfly/react-virtualized-extension/v/4.5.15)
+## 2020.09 release notes (2020-07-17)
+- [@patternfly/react-catalog-view-extension@4.5.1](https://www.npmjs.com/package/@patternfly/react-catalog-view-extension/v/4.5.1)
+- [@patternfly/react-charts@6.6.6](https://www.npmjs.com/package/@patternfly/react-charts/v/6.6.0)
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:9]: Link is correct but text needs to be updated to @6.6.0

File: packages/react-docs/RELEASE-NOTES.md [hunk 5]
```diff
@@ -35,23 +35,35 @@ releaseNoteTOC: true
...
+- **FormSelect:** 
+  - Added warning state ([#4554](https://github.com/patternfly/patternfly-react/pull/4554))
 - **Options menu:** 
   - Fixed accessibility bug in plain with text disabled part ([#4424](https://github.com/patternfly/patternfly-react/pull/4424))
 - **Page header tools:**
   - Updated to make page header tools item visibility props optional ([#4486](https://github.com/patternfly/patternfly-react/pull/4486))
 - **Popover:**
-  -  Rewrote tooltip/popover to use popper ([#4491](https://github.com/patternfly/patternfly-react/pull/4491))
+  - Rewrote tooltip/popover to use popper ([#4491](https://github.com/patternfly/patternfly-react/pull/4491))
+  - Added tippyProps back so as not to cause breaking changes ([#4543]
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:48]: Add link https://github.com/patternfly/patternfly-react/pull/4543

File: packages/react-docs/RELEASE-NOTES.md [hunk 6]
```diff
@@ -35,23 +35,35 @@ releaseNoteTOC: true
...
+  - Added warning state ([#4554](https://github.com/patternfly/patternfly-react/pull/4554))
+- **TextInput:** 
+  - Added warning state ([#4554](https://github.com/patternfly/patternfly-react/pull/4554))
 - **ToolBar:** 
   - Used alignment prop instead of align in ToolbarItem ([#4481](https://github.com/patternfly/patternfly-react/pull/4481))
   - Updated docs to use overflow menu components ([#4459](https://github.com/patternfly/patternfly-react/pull/4459))
 - **Tooltip:**
-  -  Rewrote tooltip/popover to use popper ([#4491](https://github.com/patternfly/patternfly-react/pull/4491))
+  - Rewrote tooltip/popover to use popper ([#4491](https://github.com/patternfly/patternfly-react/pull/4491))
+  - Added tippyProps back so as not to cause breaking changes ([#4543]
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:65]: Add link https://github.com/patternfly/patternfly-react/pull/4543

File: packages/react-docs/RELEASE-NOTES.md (hunk 3 above)
Comment (evwilkin) [reply packages/react-docs/RELEASE-NOTES.md:176]: @tlabaj it looks like this is still showing under 2020.08

**Pull request merged**